### PR TITLE
Add boxel dependency and necessary config and deps to scheduled-payments-client

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

File: packages/scheduled-payments-client/ember-cli-build.js [hunk 1]
```diff
@@ -6,11 +6,73 @@ process.env.EMBROIDER_REBUILD_ADDONS = '@cardstack/boxel';
...
+          },
+          {
+            prefixIds: false,
+          },
+          {
+            uniqueID: svgoUniqueId,
+          },
+        ],
+      },
+    },
```
Comment (danibonilha) [inline packages/scheduled-payments-client/ember-cli-build.js:33]: I've tried to add most of the boxel stuff, but the requirements were not clear, maybe we should think about documenting it, or create some sort of template.

File: packages/scheduled-payments-client/package.json [hunk 2]
```diff
@@ -70,11 +72,13 @@
     "ember-cli-typescript": "^5.1.1",
     "ember-cli-typescript-blueprints": "^3.0.0",
     "ember-fetch": "^8.1.1",
+    "ember-focus-trap": "^1.0.1",
```
Comment (danibonilha) [inline packages/scheduled-payments-client/package.json:75]: I don't know if there's some ember limitation, but shouldn't boxel, handled all required deps in it's package.json, so we don't need to individually add the package on each client ?

File: packages/scheduled-payments-client/app/templates/application.hbs [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {{page-title 'ScheduledPaymentsClient'}}
 
-{{! TODO: Remove once we have components }}
-<GlintTestComponent />
```
Comment (danibonilha) [inline packages/scheduled-payments-client/app/templates/application.hbs:4]: we can delete this from components folder then

**Review by danibonilha — APPROVED** [review]

File: packages/scheduled-payments-client/package.json (hunk 2 above)
Comment (lukemelia) [reply packages/scheduled-payments-client/package.json:75]: You are generally right. There is a funky embroider/addon bug that necessitates this.

Comment (lukemelia) [reply packages/scheduled-payments-client/package.json:75]: https://github.com/embroider-build/embroider/issues/1175

File: packages/scheduled-payments-client/ember-cli-build.js (hunk 1 above)
Comment (lukemelia) [reply packages/scheduled-payments-client/ember-cli-build.js:33]: Agreed. It is pretty esoteric right now, mostly with the webpack processing requirements.

**Pull request merged**